templates/model.txt

```diff
@@ -9,9 +9,13 @@ const schema = mongoose.Schema({<% _.each(fields, (field) => { %>
 
 module.exports = mongoose.model('<%= table %>', schema, '<%= table %>');
 <% } else {
+%><%
+const pascalCase = function(input) {
+  return _.chain(input).camelCase().upperFirst().value();
```

Comment:
[Code style] This logic should not be defined in the template, but in the services that call the template (`dumper.js` & `migrator.js`) to make the template silly.
It will also make the template easier to read.